### ISSUE TITLE
Investigate low accuracy in OpenPose V2

### ISSUE THREAD

**Issue by amalbasaTT** [tenstorrent/pytorch2.0_ttnn#782]
https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/docs/models/OpenPose%20V2/input_variations.md

**Comment by amalbasaTT** [comment]
Every operation needs to have PCC of 1.0 in order for this model to have a high accuracy (of 1.0). 
When fallbacking ops [conv2d, batch_norm, elu, relu], accuracy is 0.931.
When fallbacking ops [conv2d, batch_norm, elu, relu, add], accuracy is 1.0.
